frontend/src/pages/RunList.test.tsx

```diff
@@ -61,76 +84,86 @@ describe('RunList', () => {
     expect(tree).toMatchSnapshot();
   });
 
-  it('calls error callback when loading runs fails', async () => {
-    (Apis as any).runServiceApi = {
-      getRun: jest.fn(),
-      listRuns: () => Promise.reject('bad stuff'),
-    };
+  it('reloads the run when refresh is called', async () => {
+    mockNRuns(0, {});
+    const props = generateProps();
+    const tree = TestUtils.mountWithRouter(<RunList {...props} />);
+    await (tree.instance() as RunList).refresh();
+    expect(Apis.runServiceApi.listRuns).toHaveBeenLastCalledWith('', 10, RunSortKeys.CREATED_AT + ' desc', undefined, undefined);
```

Comment:
This should confirm that it was called an additional time when `refresh()` was called